lib/LSM9DS1/LSM9DS1.cpp

```diff
@@ -105,6 +110,18 @@ void LSM9DS1::readGyroscope(float &x, float &y, float &z) {
 }
 
 
+void LSM9DS1::setGyroActivity(bool wake) {
```

Comment:
Review this method and the registers used, please add clarifying comments on the register in the header file